frontend/src/components/widgets/DataGrid/DataGridCells.tsx

```diff
@@ -34,6 +35,32 @@ function getDisplayContent(quiverCell: DataFrameCell): string {
   )
 }
 
+/**
+ * Extracts a CSS property value from a given CSS style string by using a regex.
+ *
+ * @param cssId: The css ID of the element to extract the property for.
+ * @param cssStyle: The css style string.
+ * @param property: The css property to extract the value for.
+ *
+ * @return the CSS property value or undefined if the property is not found.
+ */
+function extractCssProperty(
+  cssId: string,
+  cssStyle: string,
+  property: string
+): string | undefined {
+  const regex = new RegExp(
```

Comment:
Maybe a comment on what this regex does?

Reply:
Yep, that sounds good. I added a comment and additional unit tests specific to this function.